internal/langserver/handlers/execute_command.go

```diff
@@ -12,10 +12,11 @@ import (
 )
 
 var handlers = cmd.Handlers{
-	cmd.Name("rootmodules"):        command.ModulesHandler,
-	cmd.Name("module.callers"):     command.ModuleCallersHandler,
-	cmd.Name("terraform.init"):     command.TerraformInitHandler,
-	cmd.Name("terraform.validate"): command.TerraformValidateHandler,
+	cmd.Name("rootmodules"):          command.ModulesHandler,
+	cmd.Name("module.callers"):       command.ModuleCallersHandler,
+	cmd.Name("terraform.init"):       command.TerraformInitHandler,
+	cmd.Name("terraform.validate"):   command.TerraformValidateHandler,
+	cmd.Name("terraform.modulelist"): command.ModuleListHandler,
```

Comment:
```suggestion
	cmd.Name("module.calls"): command.ModuleCallsHandler,
```
We only prefix the actual commands with `terraform`.